content/blog/2023/web-pwa-native-RWA1.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Part 1 - Responsive Web App vs PWA vs Native Mobile App
+sub: When to choose a responsive web app (RWA) for your digital product
+author: sondra
+date: 2023-05-05
+tags:
+  - Article
+  - Business Development
+  - Push Notifications
+  - Recommendations
+  - Startups
+  - User Experience
+image:
+  src: blog/2023/part1-web.jpg
+  alt: |
+    spider web with dew drops
+summary: |
+  The decision of what platform
+  to build your app with
+  is quite important –
+  affecting project scope, timeline, and budget.
+  But understanding the differences between
+  a responsive web app,
+  a progressive web app,
+  or a native mobile app,
+  and which one is right for your project
+  can seem insurmountably difficult.
+  Welcome to part 1 of a three-part series
+  unpacking the reasons to choose
+  one platform over another.
+  Let's start with the responsive web app (RWA).
+---
+{% import 'embed.macros.njk' as embed %}
+
+You have an idea for a custom digital product you’d like to build, and it’s time to decide how to build it. One of your friends swears you won’t succeed unless you build a native mobile app for iPhone. A colleague argues that a responsive web app is the only way to go. Who’s right? Are there alternative options? The decision of how to build your app is quite important – affecting project scope, timeline, and budget – but understanding when to choose one option over another can seem insurmountably difficult.
+
+The great news is that you don’t need to spend years becoming a web developer yourself in order to make a wise choice. In the following three-part series, we’ll dig into the three main options for building mobile apps: Responsive Web Apps (RWAs), Progressive Web Apps (PWAs), and Native Mobile Apps, and how to determine the best option for your project. As you read, think about who your app will serve, what their most pressing needs are, and how your app will meet those needs.
+
+## What are RWAs, PWAs, and Native Mobile Apps?
+Here are the biggest differences in a nutshell.
+
+- **Responsive Web App (RWA)** - An application that runs in a browser, such as Chrome, Firefox, or Safari, and responds to fit any screen size on any device.
+- **Progressive Web App (PWA)** - A responsive web app with additional capabilities that mimic a native mobile app.
+- **Native Mobile App** - An app built specifically for one particular mobile operating system, like iOS or Android.
+
+
+{{ embed.img(
+  src='blog/2023/web-pwa-native-comparison-chart.png',
```

Comment:
Agreed @jerivas, that is the long-term goal but in order to get this out the door in a timely manner, Sondra went with this image. At standup we spoke about maybe having the picture element allow more art direction since this chart will be unreadable on smaller screens so if we could at least do the following in the short term, it would act as a bandage:

  1. Add a long description ... although the longDesc is deprecated, I think [Mozilla recommends](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/longDesc#usage_notes) adding a link on the image itself to a text-based alternative page
  2. See if we can have the picture element use a stacked version of this chart for smaller screens.
  
  I think both of these will be faster than making this chart responsive, but I could be convinced otherwise. :)